autosubmit/autosubmit.py

```diff
@@ -691,6 +697,13 @@ def parse_args() -> Tuple[int, Optional[argparse.Namespace]]:
                                help='Stop all current running autosubmit processes, will ask for confirmation')
             group.add_argument('-fa', '--force_all', default=False, action='store_true',
                                help='Stop all current running autosubmit processes')
+            subparser.add_argument(
+                "-y",
+                "--yes",
+                default=False,
+                action="store_true",
+                help="Automatically answer yes to prompts",
```

Comment:
Hmmm, some help statements start in upper case, while others in lower case. Noted, will add an issue for that later.